project_stock_portal/README.rst

```diff
@@ -74,10 +74,6 @@ Maintainer
 
 This module is maintained by Sygel.
 
-.. image:: https://pbs.twimg.com/profile_images/702799639855157248/ujffk9GL_200x200.png
```

Comment:
@manuelregidor  y este link @Jaimermaccione ??? 
tiene que aparecer el de sygel. Por favor, cambiarlo y no solo lo borréis.